### PR TITLE
[CMOB-1397]internet handling

### PR DESCRIPTION
**Pull request by mariammnamgaladze**
https://spaceneobank.atlassian.net/browse/CMOB-1397

### CODE REVIEW

**Review by GiorgiMamagulashvil1 — APPROVED** [review]

**Review by Gio481 — APPROVED** [review]

File: app/src/main/java/com/space/spacechatapp/common/extension/InternetExtension.kt [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    return if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
+        val capabilities =
+            connectivityManager.getNetworkCapabilities(connectivityManager.activeNetwork)
+        capabilities?.run {
+            hasTransport(NetworkCapabilities.TRANSPORT_CELLULAR) || hasTransport(
+                NetworkCapabilities.TRANSPORT_WIFI
+            ) || hasTransport(NetworkCapabilities.TRANSPORT_ETHERNET)
+        } ?: false
+    } else {
+        return false
```
Comment (GiorgiMamagulashvil1) [inline app/src/main/java/com/space/spacechatapp/common/extension/InternetExtension.kt:21]: remove second return

File: app/build.gradle [hunk 2]
```diff
@@ -11,7 +11,7 @@ android {
 
     defaultConfig {
         applicationId "com.example.spacechatapp"
-        minSdk 26
+        minSdk 24
```
Comment (AleksandreGalogreBaramidze) [inline app/build.gradle:14]: ?

File: app/src/main/java/com/space/spacechatapp/presentation/chat/fragment/ChatFragment.kt [hunk 3]
```diff
@@ -34,17 +37,26 @@ class ChatFragment : BaseFragment<ChatViewModel>(R.layout.fragment_chat) {
         }
     }
 
+    private fun filterMessages(messages: List<MessageModel>): List<MessageModel> {
+        return messages.filter {
+            it.user == adapterListener.invoke() || it.isOnline
+        }
+    }
```
[the remark this answers is not in the document]
Comment (AleksandreGalogreBaramidze) [reply app/src/main/java/com/space/spacechatapp/presentation/chat/fragment/ChatFragment.kt:44]: @mariammnamgaladze gaaswore es da mec daavaaproveb

[the remark this answers is not in the document]
Comment (mariammnamgaladze) [reply app/src/main/java/com/space/spacechatapp/presentation/chat/fragment/ChatFragment.kt]: done

File: app/src/main/java/com/space/spacechatapp/common/extension/InternetExtension.kt (hunk 1 above)
Comment (mariammnamgaladze) [reply app/src/main/java/com/space/spacechatapp/common/extension/InternetExtension.kt]: done

File: app/build.gradle (hunk 2 above)
Comment (mariammnamgaladze) [reply app/build.gradle:14]: ragacas vtestavdi da shevcvale

**Review by GiorgiMamagulashvil1 — APPROVED** [review]

**Pull request merged**